test/albums.js

```diff
@@ -0,0 +1,73 @@
+const chai = require('chai'),
+  server = require('./../app'),
+  config = require('../config'),
+  nock = require('nock'),
+  expect = chai.expect;
+
+const correctUser = {
+  first_name: 'Carlos',
+  last_name: 'Bollero',
+  email: 'carlos@wolox.com.ar',
+  password: 'abcd1234'
+};
+
+const createUser = user =>
+  chai
+    .request(server)
+    .post('/users')
+    .send(user);
+
+const logIn = ({ email, password }) =>
+  chai
+    .request(server)
+    .post('/users/sessions')
+    .send({ email, password });
+
+const albums = [
+  {
+    userId: 1,
+    id: 1,
+    title: 'quidem molestiae enim'
+  },
+  {
+    userId: 1,
+    id: 2,
+    title: 'sunt qui excepturi placeat culpa'
+  },
+  {
+    userId: 1,
+    id: 3,
+    title: 'omnis laborum odio'
+  }
+];
+
+describe('/albums GET', () => {
+  it('should successfully get the albums list when logged in', () => {
+    nock('albums.com')
+      .get('/')
+      .reply(200, { albums });
+    return createUser(correctUser).then(() =>
+      logIn(correctUser).then(({ body }) =>
+        chai
+          .request(server)
+          .get('/albums')
+          .set({ token: body.token })
```

Comment:
i don't know if this works. We send the headers in this format `.set(header, value)`, for example `.set('Authorization', token)`

Reply:
It worked for previous login tests, i'll try it though